src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -70,5 +85,7 @@ public RobotMap() {
         HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
 		HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.joystick);
 
+        Component.intakeDrawbridgeSolenoid = new SolenoidSubsystem("Intake Drawbridge Solenoid", false, SolenoidState.RETRACT, Port.Pneumatics.DRAWBRIDGE_INTAKE_SOLENOID.buildDoubleSolenoid()); //TODO: check if CANTalonFX or SRX
+        Component.intakeAxleMotor = new Motor("Intake Motor", false, new CANTalonFX(Port.CANMotor.AXLE_INTAKE_MOTOR)); //TODO: check if CANTalonFX or SRX
```

Comment:
The `Component.intake` object is never constructed.

Reply:
Also Intake never seems to be used directly? Is it strictly necessary or can we remove it entirely (especially if neither the solenoid nor the motor ever need to move in tandem)?